src/models/User.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * User class.
+ */
+class User {
+  /**
+   * Default constructor.
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.firstname] - Template to retrive firstname in user.
```

Comment:
for all, not a template its the user firstname.

Reply:
done